### PR TITLE
July 25, 2024 Meeting

### PR DESCRIPTION
**Pull request by tkuhrt**

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

**Review by petermetz — APPROVED** [review]

File: meeting-minutes/2024/2024-07-25-TOC-meeting-record.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- 2024 Q3 Hyperledger Indy (due August 1, 2024)
+- [2024 TOC Project Update Calendar](../../project-reports/2024/2024-updates.md)
+
+# Discussion
+- [CommunityOne Bot](https://communityone.io/) - Fay
+- [Infrastructure changes for LF Decentralized Trust](https://docs.google.com/document/d/1E3cYIGIckXyI2h74Ee7_DCxi48PQmJ38rCnbnSZ9Clg/edit)
+- Task force: OpenSSF Scorecard
+
+# Recordings
+- [Recordings are available on the Hyperledger calendar](https://zoom-lfx.platform.linuxfoundation.org/meetings/hyp)
```
Comment (denyeart) [inline meeting-minutes/2024/2024-07-25-TOC-meeting-record.md]: @ryjones 
I've noticed this link is broken lately, it returns "Project Not Found". Where are the recordings stored now?

Comment (tkuhrt) [reply meeting-minutes/2024/2024-07-25-TOC-meeting-record.md]: That's new. You should be able to go to https://openprofile.dev/my-meetings to see any meetings that you attended.

Comment (ryjones) [reply meeting-minutes/2024/2024-07-25-TOC-meeting-record.md]: @denyeart @tkuhrt I've fixed the link. The slug change broke it.

Comment (denyeart) [reply meeting-minutes/2024/2024-07-25-TOC-meeting-record.md]: Thanks, works now!

**Review by denyeart — APPROVED** [review]

**Pull request merged**